src/Robo/Commands/Setup/BuildCommand.php

```diff
@@ -134,4 +134,45 @@ public function installAlias() {
       ->run();
   }
 
+  /**
+   * Checks the repo if there is modified files.
+   */
+  public function checkCleanRepo() {
```

Comment:
Because this is public, it will become a command automatically. I don't think we want that, so make it protected.